app/src/main/res/layout-land/story_chapter_view.xml

```diff
@@ -101,7 +102,10 @@
     <FrameLayout
       android:layout_width="match_parent"
       android:layout_height="match_parent"
-      android:layout_margin="16dp"
+      android:layout_marginStart="76dp"
```

Comment:
Below is dummy text added for testing purpose.
![2020-03-12](https://user-images.githubusercontent.com/54615666/76507328-a2e22380-6472-11ea-8adf-18a63c48e485.png)
widget id chapter_summary line number 75 handle the margin bottom, chapter_summary edge is cutting when summary is longer.

Reply:
This has been resolved.

![Screenshot_1584116428](https://user-images.githubusercontent.com/9396084/76639917-0c4a5b00-6575-11ea-8940-9c1899bc559e.png)
![Screenshot_1584116430](https://user-images.githubusercontent.com/9396084/76639922-0f454b80-6575-11ea-9bb1-da3312541546.png)